js/components/accessibilityControls.js

```diff
@@ -61,6 +73,41 @@ AccessibilityControls.prototype = {
     }
   },
 
+  keyEventUp: function(e) {
+    if (!this.controller.state.accessibilityControlsEnabled) { return; }
+    var targetTagName = this.getTargetTagName(e);
+    var charCode = e.which || e.keyCode;
+    this.handleKeyClick(e, this.state.keyMapsList, charCode, false, targetTagName);  //stop rotate 360
+  },
+
+  getTargetTagName: function(e) {
+    var targetTagName = "";
+    if (e.target && typeof e.target.tagName === "string") {
+      targetTagName = e.target.tagName.toLowerCase();
+    }
+    return targetTagName;
+  },
+
+  handleKeyClick: function(e, keyMapsList, char, bool, targetTagName) {
```

Comment:
Can we rename this to something more descriptive? It looks like this is only used for VR, so it might be better to name it according to its purpose. The `e`, `char` and `bool` parameters are also not very descriptive.

Reply:
\+